recipe/install-r-xgboost.bat

```diff
@@ -2,10 +2,14 @@
 mkdir "%SRC_DIR%"\build-r
 pushd "%SRC_DIR%"\build-r
 
+set PATH=%LIBRARY_PREFIX%\bin;%PATH%
+
 cmake -G "Ninja" ^
     -DCMAKE_BUILD_TYPE:STRING="Release" ^
     -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
     -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON ^
+    -DDLLTOOL_EXE=%BUILD_PREFIX%\Library\bin\%DLLTOOL% ^
+    -DLIBR_CORE_LIBRARY=%PREFIX%\lib\R\bin\x64\R.lib ^
```

Comment:
Could you please add a comment here? It would help future readers understand when this may or may not be needed (particularly for other R recipes)

Reply:
FYI, it's not possible to add comments within the command that's broken across several lines. It can only be done before/after

Reply:
Adding before would be ok

Reply:
I also had to add this because `cmake` was choosing the wrong value here.  We can try removing it, and I'll at least comment explaining this if it is still needed.